### PR TITLE
ArduCopter:Airframe: Update to QGC3.5

### PR DESCRIPTION
**Pull request by hamishwillee**
This is my guess at how you set frame type for ArduCopter in QGC3.5. The text in the dialog might be wrong as there is no way for user to explicitly load params.

That text should perhaps instead say

> Airframe Setup is used to select the airframe class and type that matches your vehicle, and will load the default parameters for that type.

@DonLakeFlyer Can you confirm this is correct/approve

### CODE REVIEW

**Comment by DonLakeFlyer** [comment]
Can you attach an image of what it looks like now?

**Comment by hamishwillee** [comment]
@DonLakeFlyer Sorry, missed this one

> Can you attach an image of what it looks like now?

Yes, so it looks like this:



and I am suggesting that text might be better as

> Airframe Setup is used to select the airframe class and type that matches your vehicle, and will load the default parameters for that type.

**Comment by DonLakeFlyer** [comment]
This text is wrong. Changing the frame has no affect on parameter values other than the frame parameter. I'll fix it. It's a holdover from whenm there was a list of parameter sets you could choose from in this page. That's gone now.

**Comment by DonLakeFlyer** [comment]
Should just be: "Airframe Setup is used to select the airframe class and type that matches your vehicle."

File: en/SetupView/airframe_ardupilot.md [hunk 1]
```diff
@@ -9,15 +9,19 @@ This page allows you to configure the main airframe selection associated with yo
...
 
-   * You can also click **Load common parameters** to load a parameter set for known vehicle types.
+   ![Airframe type](../../assets/setup/airframe/arducopter_class.jpg)
+   
+   > **Note** You will need to reboot the vehicle for class changes to take effect.
+1. Select the specific *Frame Type* for your vehicle:
+
+   ![Airframe type](../../assets/setup/airframe/arducopter_type.jpg)
+
+On selection *QGroundControl* will automatically upload the appropriate parameters for the vehicle class/type.
```
Comment (DonLakeFlyer) [inline en/SetupView/airframe_ardupilot.md]: QGC does not do this.

Comment (hamishwillee) [reply en/SetupView/airframe_ardupilot.md]: OK. Removed.

**Comment by hamishwillee** [comment]
@DonLakeFlyer Thanks. I've updated as you suggested and will squash and merge. As discussed, you're fixing the text for ArduPilot airframe selection to remove comment about parameters. I'm happy to merge because those fixes will appear in next release/daily, not in current stable release.

**Pull request merged**